plugins/services/src/js/components/forms/VolumesFormSection.js

```diff
@@ -345,14 +348,32 @@ class VolumesFormSection extends Component {
                 volume={volume}
                 index={key}
               >
-                <FieldSelect name={`volumes.${key}.type`} value={volume.type}>
-                  <option value="">Select...</option>
-                  <option value="HOST">
-                    Host Volume
-                  </option>
-                  <option value="PERSISTENT">Persistent Volume</option>
-                  <option value="EXTERNAL">External Volume</option>
-                </FieldSelect>
+                <Select
+                  name={`volumes.${key}.type`}
+                  value={volume.type}
+                  placeholder="Select ..."
+                >
+                  {Object.keys(VolumeDefinitions).map((type, index) => {
+                    if (type === "DSS" || type === "EPHEMERAL") {
+                      return <noscript />;
```

Comment:
We use this to not show certain types, if I understand this correctly. In that case I would like to see a filter instead, because it took me a while reading this part to understand what it does exactly. In my head this should also work and IMHO it reads a bit clearer.

```js
const excludedTypes = ["DSS", "EPHERMAL"];
Object.keys(VolumeDefinitions)
    .filter(type => !excludedTypes.includes(type))
    .map((type, index) => {
```

This is no show stopper for me, especially because the other PR has the same solution and I really value consistency 👍

Reply:
I like that change too. it's clearer.

I will change and test again. Thanks